engine/installation/linux/docker-ee/ubuntu.md

```diff
@@ -156,6 +157,16 @@ from the repository.
        stable-{{ site.docker_ee_version }}"
     ```
 
+    **ppc64le**:
+
+    ```bash
+    $ sudo add-apt-repository \
+       "deb [arch=ppc64le] {{ download-url-base }} \
```

Comment:
most repositories actually use the historic name `ppc64el`. While I don't think I have access to the docker-ee link, this is how the docker-ce repo is setup. (ppc64le won't work)

Reply:
@seemethere @andrewhsu ping ^
What I see on the store side is `le` - can you confirm this is what we did? and also that it is consistent with the CE side?

Reply:
Adding color: it's currently `el` for Ubuntu, but `le` for SLES (have not checked the others).

Reply:
@andrewhsu @seemethere ^

Reply:
ubuntu uses `ppc64el`: https://wiki.ubuntu.com/ppc64el
so docker's ubuntu repo was setup as such: https://download.docker.com/linux/ubuntu/dists/xenial/pool/edge/ppc64el/